### PR TITLE
Cq-branch-27-april

### PR DESCRIPTION
**Pull request by deonvisser44**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -39,70 +51,111 @@ function prepareDataForMap (results: UnprocessedResultsFromCRM[], searchAddressP
...
         if (isReadyForSearch) {
             const getDataFromCrm = async () => {
                 setLoading(true)
-                const searchAddressPosition = await getSearchAddressPosition(searchParameters)
-                const { matchedProperties, numberOfUniqueSearchRecords } = await findMatchingRecords(searchParameters, filterInUse, searchAddressPosition)
+                const searchAddressPosition = await getSearchAddressPosition(
+                    searchParameters
+                )
+                const { matchedProperties, numberOfUniqueSearchRecords } =
+          await findMatchingRecords(
```
Comment (jeznag) [inline src/App.tsx]: indentation is off here

File: src/App.tsx [hunk 2]
```diff
@@ -123,15 +176,52 @@ function App () {
         void getMapsAPIKeyFromCRM()
     }, [])
 
+    function toggleMapSize (mapFullScreen: boolean) {
```
Comment (jeznag) [inline src/App.tsx:135]: can we go with `isMapFullScreen`? Prefer is/has prefix for boolean vars: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/boolean-prop-naming.md

File: src/App.tsx [hunk 3]
```diff
@@ -123,15 +176,52 @@ function App () {
...
+                        setReadyForSearch={setReadyForSearch}
+                        setFilterInUse={setFilterInUse}
+                        filterInUse={filterInUse}
+                        updateResults={updateResults}
+                        mapFullScreen={mapFullScreen}
+                    />
+                    <div className={mapFullScreen ? 'map-div-fw' : 'map-div'}>
+                        {isLoading && <p>Loading map...</p>}
+                        {searchAddressPosition &&
+              renderMapWidget(
```
Comment (jeznag) [inline src/App.tsx]: indentation

File: src/components/LeasesSearch.tsx [hunk 4]
```diff
@@ -1,106 +1,253 @@
...
+                        onChange={(e: ChangeEvent<HTMLInputElement>) => {
+                            props.changeSearchParameters({
+                                ...props.searchParameters,
+                                landArea: {
+                                    min: e.target.valueAsNumber || -1,
+                                    max: props.searchParameters.landArea.max
+                                }
+                            })
+                        }}
+                        placeholder="Min"
```
Comment (jeznag) [inline src/components/LeasesSearch.tsx]: what were the changes in this file for?

File: src/components/SalesEvidenceSearchWidget.tsx [hunk 5]
```diff
@@ -1,84 +1,192 @@
...
+                changeSearchParameters={props.changeSearchParameters}
+                changeSearchParametersForm={props.changeSearchParametersForm}
+                searchParameters={props.searchParameters}
+                setFilterInUse={props.setFilterInUse}
+                filterInUse={props.filterInUse}
+                updateResults={props.updateResults}
+            />
+            <form className="search-widget-form">
+                <label className="eight">
+          Land Area m<sup>2</sup>
```
Comment (jeznag) [inline src/components/SalesEvidenceSearchWidget.tsx]: indentation

File: src/components/SalesEvidenceSearchWidget.tsx [hunk 6]
```diff
@@ -1,84 +1,192 @@
...
-                        })
-                    }} placeholder="Min" type="number" tabIndex={4} />
-                    <input className="minMaxSize border" id="buildAreaMax" onChange={(e: ChangeEvent<HTMLInputElement>) => {
-                        props.changeSearchParameters({
-                            ...props.searchParameters,
-                            buildArea: { min: props.searchParameters.buildArea.min, max: e.target.valueAsNumber || -1 }
-                        })
-                    }} placeholder="Max" type="number" tabIndex={4} />
+                <label className="nine">
+          Build Area m<sup>2</sup> <br />
```
Comment (jeznag) [inline src/components/SalesEvidenceSearchWidget.tsx]: indentation

File: src/components/SalesEvidenceSearchWidget.tsx [hunk 7]
```diff
@@ -1,84 +1,192 @@
...
+                setReadyForSearch={props.setReadyForSearch}
+                changeSearchParameters={props.changeSearchParameters}
+                changeSearchParametersForm={props.changeSearchParametersForm}
+                searchParameters={props.searchParameters}
+                setFilterInUse={props.setFilterInUse}
+                filterInUse={props.filterInUse}
+                updateResults={props.updateResults}
+            />
+            <form className="search-widget-form">
+                <label className="eight">
```
Comment (jeznag) [inline src/components/SalesEvidenceSearchWidget.tsx]: are these changes expected?

**Comment by jeznag** [comment]
@deonvisser44 can you resolve the conflicts in yarn.lock and then we're good to merge?

**Comment by deonvisser44** [comment]
Hi Jeremy, in an attempt to correct the formatting I installed the npm
package called standard after eslint didn't fix it, it didn't work to fix
the formatting so I ran npm uninstall to remove it. It then showed that
yarn.lock was modified even though I uninstalled it, I thought that since I
installed and uninstalled a package the change wouldn't have to be
committed. So I only committed the changes to the files I manually changed
the format of. Could this be the issue? Would replacing the yarn.lock file
with the original from main solve this?

**Pull request merged**